code/modules/client/asset_cache.dm

```diff
@@ -191,6 +191,7 @@ You can set verify to TRUE if you want send() to sleep until the client has the
 		"large_stamp-cent.png"      = 'icons/paper_icons/large_stamp-cent.png',
 		"large_stamp-syndicate.png" = 'icons/paper_icons/large_stamp-syndicate.png',
 		"large_stamp-rep.png"	    = 'icons/paper_icons/large_stamp-rep.png',
+		"large_stamp-magistrate.png"= 'icons/paper_icons/large_stamp-magistrate.png',
```

Comment:
No spess behind `=`? :thinking:

Reply:
@KasparoVy the intent was to keep the equal sign there aligned with the other ones.